### PR TITLE
Us 41 remove or disable user

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

File: src/main/java/dao/PersonDao.java [hunk 1]
```diff
@@ -30,5 +30,9 @@ public void deleteById(Long id) {
         return namedQuery("Person.findAll").list();
     }
 
+    public List<Person> findAllForManaging(){
+        return namedQuery("Person.findAllForManaging").list();
```
Comment (drauf) [inline src/main/java/dao/PersonDao.java]: maybe name it `findAll` and rename the original `findAll` to `findAllActive`?

File: src/main/java/resources/PersonResource.java [hunk 2]
```diff
@@ -72,6 +72,13 @@ public List fetchPersonsWithSlots(@QueryParam("startDate") String startDate, @Qu
     }
 
     @GET
+    @Path("/all/withoutSlots")
+    @UnitOfWork
+    public List fetchPersonsWithoutSlots() {
```
Comment (danielr-kainos) [inline src/main/java/resources/PersonResource.java]: is this named correctly? we are not fetching persons without slots, but all persons

File: src/main/java/resources/PersonResource.java [hunk 3]
```diff
@@ -72,6 +72,13 @@ public List fetchPersonsWithSlots(@QueryParam("startDate") String startDate, @Qu
...
+        return personDao.findAll();
```
Comment (rlewan) [inline src/main/java/resources/PersonResource.java:78]: Shouldn't this call `findAllActive`?

Comment (r-kruk-kainos) [inline src/main/java/resources/PersonResource.java:78]: It's called for fetching all the persons for managing, even the disabled ones

File: src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-manage-users.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+ <h3 class="center" id="manage-users-header">Manage users</h3>
```
Comment (scyganska) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-manage-users.html]: Views should be in html/partials

File: src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-table.html [hunk 5]
```diff
@@ -81,4 +81,6 @@ <h1 id="recruiter-header">Availabilities for interviews</h1>
```
Comment (scyganska) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-table.html]: Views should be in html/partials

File: src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-manage-users.html (hunk 4 above)
Comment (rlewan) [reply src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-manage-users.html]: +1

File: src/main/resources/assets/js/application/recruiter/controllers/tdprManageUsersController.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        tdprPersonsService.fetchPersons().then( function (response) {
+            $scope.persons = response.data;
+        }, function () {
+            Notification.error({message: "Something went wrong with getting users" , delay: 3500});
+        });
+
+        $scope.managePerson = function( person ){
+            person.active = !person.active;
+            tdprPersonsService.managePerson(person).then( function () {
+                if(!person.active) Notification.success({message: person.firstName + " " + person.lastName  + " has been disabled." , delay: 3500});
```
Comment (rlewan) [inline src/main/resources/assets/js/application/recruiter/controllers/tdprManageUsersController.js]: Please use `{ }` blocks in if statements for readability.

**Comment by rlewan** [comment]
Users in the table should be sorted alphabetically (can be name first).

**Comment by RogowskaMalgorzata** [comment]
Looks good.

**Pull request merged**